usecases/config/environment.go

```diff
@@ -372,6 +372,10 @@ func FromEnv(config *Config) error {
 		return err
 	}
 
+	if v := os.Getenv("REPLICATION_FORCE_OBJECT_DELETION_CONFLICT_RESOLUTION"); v != "" {
```

Comment:
maybe a better name would be `ENABLE_REPLICATION_FORCE_OBJECT_DELETION_CONFLICT_RESOLUTION`? bc this that kind of setting which enables or disables some kind of behaviour, rigt? if yes then you can use `entcfg.Enabled(os.Getenv("REPLICATION_FORCE_OBJECT_DELETION_CONFLICT_RESOLUTION"))`

Reply:
aham, it's actually the string value with the forced replication strategy